libraries/SITL/SIM_Sailboat.h

```diff
@@ -0,0 +1,56 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  sailboat simulator class
+*/
+
+#pragma once
+
+#include "SIM_Aircraft.h"
+
+namespace SITL {
+
+/*
+  a sailboat simulator
+ */
+class Sailboat : public Aircraft {
+public:
+    Sailboat(const char *home_str, const char *frame_str);
+
+    /* update model by one time step */
+    void update(const struct sitl_input &input);
+
+    /* static object creator */
+    static Aircraft *create(const char *home_str, const char *frame_str) {
+        return new Sailboat(home_str, frame_str);
+    }
+
+private:
+
+    void calc_lift_and_drag(float wind_speed, float angle_of_attack_deg, float& lift, float& drag);
+    float turn_circle(float steering);
+    float calc_yaw_rate(float steering, float speed);
+    float calc_lat_accel(float steering_angle, float speed);
+
+    float max_wheel_turn;
+    float turning_circle;
```

Comment:
Units/meaning/values? Very unclear what this is other then a magic gain.

Reply:
this is all existing code borrowed from the SIM_Rover class.  Still, I'll add some comments at least.